api.raml

```diff
@@ -23,28 +22,79 @@ securitySchemes:
           type: string
 
 
-#
-# Example Endpoints
-#
+/sequenceForGff3/{sequenceType}:
+  displayName: Retrieve sequence for GFF3
+  post:
+    securedBy: header-auth
+    queryParameters:
+      deflineFormat: lib.DeflineFormat
+      forceStrandedness:
+        type: boolean
+        default: false
+      basesPerLine:
+        type: integer
+        default: 60
+    body:
+      multipart/form-data:
+        type: object
+        properties:
+            file: file
+    responses:
+      200:
+        body:
+          application/octet-stream:
+      401:
+        body:
+          application/json:
+            type: error.UnauthorizedError
+      500:
+        body:
+          application/json:
+            type: error.ServerError
 
-/hello:
-  displayName: Hello World
-  get:
+/sequenceForBed/{sequenceType}:
+  displayName: Retrieve sequence for BED
+  post:
+    securedBy: header-auth
+    queryParameters:
+      deflineFormat: lib.DeflineFormat
+      forceStrandedness:
+        type: boolean
+        default: false
+      startOffset: lib.StartOffset
+      basesPerLine:
+        type: integer
+        default: 60
+    body:
+      multipart/form-data:
+        type: object
+        properties:
+            file: file
     responses:
       200:
+        body:
+          application/octet-stream:
+      401:
```

Comment:
The 401 and 500 types here are not strictly necessary.  I think they may auto-generate in the API even if they aren't here, but not totally sure.